erratatool.yml

```diff
@@ -147,7 +147,7 @@ boilerplates:
       Red Hat OpenShift Container Platform is Red Hat's cloud computing Kubernetes application platform solution designed for on-premise or private cloud deployments.
       Security Fix(es):
 
-      [[CVEs]]
+      {CVES}
```

Comment:
I think you'll want a dummy value in [group.yml](https://github.com/openshift/ocp-build-data/blob/openshift-4.10/group.yml#L3-L6) for the case where we're reading this file without calculating CVEs for a RHSA

Reply:
OTOH maybe it doesn't matter if we never try to format the contents outside the context of a RHSA - can you check? I thought it might have happened automatically at read time, but maybe it doesn't for this one.

Reply:
I searched for "vars.impact" and I didn't find anything. I don't think that var is used, we're just hardcoding it into our command.

Reply:
The boilerplate is also used when creating a new RHSA advisory (elliott create), I think that'll need to be updated. But also I don't think I've seen us use `create` for RHSAs, is this an edge case?

Reply:
I'm not sure if the create logic is updated to our other rhsa processes..and there's code duplication. I'd rather not allow this path to be sure we're not doing the wrong thing

Reply:
sorry it's `IMPACT` not `Impact`. it's referenced in the [CVE topic](https://github.com/openshift/ocp-build-data/blob/fe4e0009e3a4ada660853836af9a4ecf68b763c3/erratatool.yml#L145)

we used to create RHSAs directly since they were separate advisories; at the time we needed to create them correctly since we didn't have logic to convert between. since we moved to just converting regular advisories to RHSAs we can probably drop the create code path.